tutorial/single_pendulum/README.md

```diff
@@ -17,23 +19,15 @@
 
   The Gazebo GUI will open and the single pendulum should be already spawned in the scene.
 
-  ![Gazebo Sim GUI](media/gz_gui.png)
```

Comment:
No need to remove this, just substitute it with the version uploaded on the issue (for example https://github.com/robotology/gz-sim-yarp-plugins/assets/57228872/508f767c-9a8c-460c-87c7-9964fa01261b).